dapps/tezosDomains.tsx

```diff
@@ -0,0 +1,194 @@
+import { Parser } from "@taquito/michel-codec";
+import { Schema } from "@taquito/michelson-encoder";
+import { bytes2Char } from "@taquito/tzip16";
+import { contracts, CustomViewData, CustomView } from ".";
+import logo from "../assets/images/TezosDomains.svg";
+import Alias from "../components/Alias";
+import { transaction } from "../components/RenderProposalContentLambda";
+
+const parser = new Parser();
+
+export const COMMIT_ADDRESS = {
+  mainnet: "KT1P8n2qzJjwMPbHJfi4o8xu6Pe3gaU3u2A3",
+  ghostnet: "KT1PEnPDgGKyHvaGzWj6VJJYwobToiW2frff",
+};
+
+export const buySchema = new Schema(
+  parser.parseMichelineExpression(
+    `(pair %buy (bytes %label) (pair (nat %duration) (pair (address %owner) (pair (option %address address) (pair (map %data string bytes) (nat %nonce))))))`
+  )!
+);
+
+export const BUY_ADDRESS = {
+  mainnet: "KT191reDVKrLxU9rjTSxg53wRqj6zh8pnHgr",
+  ghostnet: "KT1Ks7BBTLLjD9PsdCboCL7fYEfq8z1mEvU1",
+};
+
+export const claimReverseRecordSchema = new Schema(
+  parser.parseMichelineExpression(
+    "(pair %claim_reverse_record (option %name bytes) (address %owner))"
+  )!
+);
+
+export const CLAIM_REVERSE_RECORD = {
+  mainnet: "KT1TnTr6b2YxSx2xUQ8Vz3MoWy771ta66yGx",
+  ghostnet: "KT1H19ouy5QwDBchKXcUw1QRFs5ZYyx1ezEJ",
+};
+
+export const tezosDomainsContracts = {
+  COMMIT_ADDRESS,
+  BUY_ADDRESS,
+  CLAIM_REVERSE_RECORD,
+};
+
+export const tezosDomainsContractsMatcher: contracts = {
+  mainnet: {
+    KT1F7JKNqwaoLzRsMio1MQC7zv3jG9dHcDdJ: true,
+    KT1QHLk1EMUA8BPH3FvRUeUmbTspmAhb7kpd: true,
+    KT1H1MqmUM4aK9i1833EBmYCCEfkbt6ZdSBc: true,
+    KT1TnTr6b2YxSx2xUQ8Vz3MoWy771ta66yGx: true,
```

Comment:
`[CLAIM_REVERSE_RECORD.mainnet]: true`?

 so for the other??

Reply:
The point is to only add const for the contracts that we want to have a custom UI, only having them in const for the keys is not useful I think

Reply:
but what about line 51?

```
 [BUY_ADDRESS.mainnet]: true,
```